selectors-4/deferred-for-level-5

```diff
@@ -127,3 +127,14 @@ The local link pseudo-class '':local-link''</h3>
 		What about if the document url has less than N segments,
 		and the link is same-page?
 		Should "null segments" count as matching, or not?
+
+<h3 id="custom-pseudo">Exposing custom state: the '':state()'' pseudo-class</h3>
+
+  The <dfn selector>:state()</dfn> pseudo-class takes an argument and matches
+  custom elements whose exposed custom states include the argument.
+
+  The grammar of the '':state()'' pseudo-class is:
+
+  <pre class=prod>:state( <<string>> )</pre>
```

Comment:
I think this wants to be 

```suggestion
  <pre class=prod>:state( [<ident> | <dashed-ident>] )</pre>
```

Reply:
Does `<ident>` not include `<dashed-ident>`?

Reply:
Oh yeah you’re right

Reply:
Should I change it to `<ident>` then? I'm not an expert on CSS grammar. @dbaron

Reply:
Yes I think it needs to be `<ident>`.

```suggestion
  <pre class=prod>:state( <ident> )</pre>
```